.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": true,
+    "singleQuote": true
```

Comment:
if we remove this line, will we have smaller diff?

Reply:
yes it will reduce the changes, do you want to me add the prettier config in a new PR

Reply:
Yes, let's remove this line